client/src/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class App extends React.Component {
+  //because I'm not using state, with more time I would probably restyle this to a functional component
+  constructor(props) {
+    super(props);
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onClick() {
+    let sheets = document.styleSheets;
+    let sheet;
+    let newCSS = document.querySelector('#css').value;
+    if (sheets.length === 0) {
+      sheet = document.createElement('style');
+      sheet.innerHTML = newCSS;
+      document.body.appendChild(sheet);
+    } else {
+      sheet = sheets[0];
+      sheet.insertRule(newCSS, sheet.cssRules.length);
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <h1 align="center">Mintbean Hackathon</h1>
+        <h2 align="center">April 4, 2020</h2>
+        <h2 align="center">Live CSS Editor</h2>
+        <div>Insert New CSS In This Box</div>
+        <textarea id='css' rows='8' cols='50'></textarea><br />
+        <button id='updater' onClick={this.onClick}>Click to Update CSS</button>

```

Comment:
Could this handler be associated with the onChange of the text area? That way the style is updated as you type without the need for the extra click.